src/components/Explorer/queryTemplate.js

```diff
@@ -1,7 +1,28 @@
+function templ(strings, value) {
+  const str0 = strings[0];
+  const str1 = strings[1];
+  if (Array.isArray(value)) {
+    const o = value.map(x => x.value);
+    const r = [];
+    if (!value.length) {
+      return '';
+    }
+    o.forEach((s) => {
+      r.push(`${str0}${s}${str1}`);
+    });
+    return `AND (${r.join(' OR ')})`;
+  }
+  return `AND ${str0}${value.value !== undefined ? value.value : value}${str1}`;
+}
+
+function validate(p) {
```

Comment:
might want to name this better, does it validate an array?  That it has at least one non-null element?